fh_immuta_utils/data_source.py

```diff
@@ -38,55 +38,23 @@
     "Amazon S3": "s3",
     "Amazon Athena": "ath",
 }
-MAX_IMMUTA_NAME_LIMIT = 55
-MAX_POSTGRES_NAME_LIMIT = 63
```

Comment:
Very nie - What are the new limits? And do we by any chance have any way to hit them?

Reply:
Immuta has told us that the new limit for the Postgres table names is 255 chars, and for the Immuta data source names it is at least 255 chars. That's the equivalent of the following string:

this_is_255_chars_this_is_255_chars_this_is_255_chars_this_is_255_chars_this_is_255_chars_this_is_255_chars_this_is_255_chars_this_is_255_chars_this_is_255_chars_this_is_255_chars_this_is_255_chars_this_is_255_chars_this_is_255_chars_this_is_255_chars_thi

I can't see any implementation having a table name longer than the above. If it does, Postgres will truncate the name itself. But I'm open to leaving the limits in there if we want to be safe and making it 255.

Reply:
Other option as suggested by @chintanms in our code review session is to state the limit in our docs

Reply:
Sweet I hope we don't run into that limit! 
I feel like if we state in our docs we might as well set the limit ourselves as well and document that?

Reply:
Sure, sounds good. I'll add the limit back in to the code and document as well

Reply:
Limits added back in with the new 255 character limit.